Gem/Code/Source/SampleComponentManager.cpp

```diff
@@ -486,6 +485,20 @@ namespace AtomSampleViewer
             }
             AZ_Warning("SampleComponentManager", targetSampleFound, "Failed find target sample %s", targetSampleName.c_str());
         }
+        if (commandLine->HasSwitch("timingSamples"))
+        {
+                AZStd::string timingSamplesStr = commandLine->GetSwitchValue("timingSamples", 0);
+                int timingSamplesCount = atoi(timingSamplesStr.c_str());
```

Comment:
There should be more robust conversion functions than raw atoi in AZ::StringFunc.

int ToInt(const char* in);
bool LooksLikeInt(const char* in, int* pInt = nullptr);

Reply:
Made this change